tests/DependencyInjection/HWIOAuthExtensionTest.php

```diff
@@ -540,13 +558,6 @@ protected function createEmptyConfiguration(): void
         $loader->load([$config], $this->containerBuilder);
     }
 
-    protected function createFullConfiguration(): void
```

Comment:
Why removing this test? It's not really related I guess.

Reply:
This is not a test. It was an unused method and actually contained some `fosub` config that does not work anymore anyway.